src/components/AddressSearch/index.js

```diff
@@ -472,17 +500,18 @@ function AddressSearch(props) {
                         }}
                         inbetweenCompo={
                             // We want to show the current location button even if there are no recent destinations
-                            props.predefinedPlaces.length === 0 && shouldShowCurrentLocationButton ? (
+                            predefinedPlaces.length === 0 && shouldShowCurrentLocationButton ? (
                                 <View style={[StyleUtils.getGoogleListViewStyle(true), styles.overflowAuto, styles.borderLeft, styles.borderRight]}>
                                     <CurrentLocationButton
                                         onPress={getCurrentLocation}
-                                        isDisabled={props.network.isOffline}
+                                        isDisabled={network.isOffline}
                                     />
                                 </View>
                             ) : (
                                 <></>
                             )
                         }
+                        placeholder=""
```

Comment:
Why are we adding a placeholder?

Reply:
It's a required property https://github.com/FaridSafi/react-native-google-places-autocomplete/blob/master/GooglePlacesAutocomplete.d.ts#L413